dashboard/lib/policies/README.md

```diff
@@ -0,0 +1,25 @@
+# Policy Objects
+
+This directory is meant to contain Plain Old Ruby Objects (POROs) that
+encapsulate business logic which _tells you about something_. This is as
+opposed to other categories of POROs like Service Objects which _do something_
+or Query Objects which _retrieve something_).
```

Comment:
This readme looks fantastic! I had several questions and all are answered here :-)